src/tools/Makefile.am

```diff
@@ -52,7 +52,7 @@ piv_tool_SOURCES = piv-tool.c util.c
 piv_tool_LDADD = $(OPTIONAL_OPENSSL_LIBS)
 opensc_explorer_SOURCES = opensc-explorer.c util.c
 opensc_explorer_LDADD = $(OPTIONAL_READLINE_LIBS)
-pkcs15_tool_SOURCES = pkcs15-tool.c util.c
+pkcs15_tool_SOURCES = pkcs15-tool.c util.c $(top_srcdir)/src/pkcs11/pkcs11-display.c $(top_srcdir)/pkcs11/pkcs11-display.h
```

Comment:
I think this needs to be added also to the windows makefiles to unbreak the appveyor build.

Reply:
Yes. I did not like how I did this and was asking @frankmorgner in https://github.com/OpenSC/OpenSC/pull/1987#issuecomment-612884325
how should be handled.  Hoping for a response, before going further.